[Layers] Error while requiring module `vuetify-nuxt-module`

Using with layers. When trying to generate, throws such error.

```
$ yarn install
yarn install v1.22.18
[1/4] Resolving packages...
success Already up-to-date.
$ nuxt prepare
[log] Nuxi 3.6.2
[info] [nuxt:tailwindcss] Using default Tailwind CSS file
[error] Error while requiring module `vuetify-nuxt-module`: Error: Cannot find module 'C:\GitLab-Runner-Front\builds\L34TKzhT\0\dit\front-vue\efo\efo\node_modules\vuetify-nuxt-module\node_modules\@nuxt\kit\dist\index.mjs'
[error] Cannot find module 'C:\GitLab-Runner-Front\builds\L34TKzhT\0\dit\front-vue\efo\efo\node_modules\vuetify-nuxt-module\node_modules\@nuxt\kit\dist\index.mjs'
  at createEsmNotFoundErr (node:internal/modules/cjs/loader:1098:15)
  at finalizeEsmResolution (node:internal/modules/cjs/loader:1091:15)
  at resolveExports (node:internal/modules/cjs/loader:567:14)
  at Module._findPath (node:internal/modules/cjs/loader:636:[31](https://gitlab.halyklife.nb/dit/front-vue/efo/-/jobs/10972#L31))
  at Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
  at Function.resolve (node:internal/modules/cjs/helpers:116:19)
  at _resolve (node_modules\jiti\dist\jiti.js:1:250100)
  at jiti (node_modules\jiti\dist\jiti.js:1:252409)
  at C:/GitLab-Runner-Front/builds/L[34](https://gitlab.halyklife.nb/dit/front-vue/efo/-/jobs/10972#L34)TKzhT/0/dit/front-vue/efo/efo/node_modules/vuetify-nuxt-module/dist/module.mjs:1:174
  at evalModule (node_modules\jiti\dist\jiti.js:1:255106)
error Command failed with exit code 1.
```

Any ideas?

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "vuetify-nuxt-module",
   "type": "module",
   "version": "0.5.4",
-  "packageManager": "pnpm@8.6.9",
+  "packageManager": "pnpm@8.6.10",
   "description": "Zero-Config Nuxt Module for Vuetify",
   "author": "userquin <userquin@gmail.com>",
   "license": "MIT",
@@ -63,15 +63,8 @@
     "prepublishOnly": "pnpm build",
     "release": "bumpp && npm publish"
   },
-  "peerDependencies": {
-    "@nuxt/kit": "^3.6.2",
-    "perfect-debounce": "^1.0.0",
-    "unconfig": "^0.3.9",
-    "vite-plugin-vuetify": "^1.0.2",
-    "vuetify": "^3.3.9"
-  },
   "dependencies": {
-    "@nuxt/kit": "^3.6.5",
+    "@nuxt/kit": "^3.6.2",
     "defu": "^6.1.2",
     "perfect-debounce": "^1.0.0",
     "unconfig": "^0.3.9",

